lib/semantics/check-call.cc

```diff
@@ -0,0 +1,475 @@
+// Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
git seems to be treating this as a new file rather than a move, so I can't tell if there are changes to review here.

Reply:
There are, it's not just a move.  I do not know why github failed to track the move on this file when it seemed to be able to do so on the other file that I moved (`check-call.h`).  I will mail you a diff.